frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/confirmation.tsx

```diff
@@ -235,7 +315,7 @@ export default function ApplyFlowConfirm() {
         <div>
           <dl className="mt-6 divide-y border-y">
             <DescriptionListItem term={t('confirm.application-code')}>
-              <strong>{submissionInfo.confirmationCode}</strong>
+              <strong>{submissionInfo?.confirmationCode ?? 'confirmationCode'}</strong>
```

Comment:
Does it make sense to render `confirmationCode` if there is no confirmation code? I feel like if the confirmation code is required to render, and it isn't defined, you should throw.

@sebastien-comeau agree/disagree?

Reply:
Normally, the loader would catch the missing submission information and throw at the loader level.

Reply:
That was what I was thinking, too.. but wasn't sure if this would align with business requirements.

Reply:
Its currently commented out in the loader and will be re-enabled once adult-child flow is completed

Reply:
Ohh.. in that case, consider adding TODOs with a comment explaining that. It's not obvious when reviewing this stuff why some things are as they are, because we don't have the full context of the changes.

And adding TODOs will add a marker in the code so we can easily see it while doing a full application review.. like this:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/cd316440-9f15-44c7-9cb9-f4864d6dfe41)

(incidentally, I don't know if anyone ever actually DOs our TODOs, considering that we have dozens of them outstanding 🤣)

Reply:
Added additional TODOs